<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>911</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 94, 0)</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>101</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/user-solid.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/user-regular.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/envelope-regular.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>500</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/eye-solid.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>610</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/calendar-regular.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>610</y>
        <width>41</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/person-half-dress-solid.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>270</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>500</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>630</y>
        <width>261</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>630</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Pictures/tt1.webp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Pictures/tt3.webp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>500</y>
        <width>261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Pictures/adb83c3bdf74362a6f65.jpg.webp</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>130</y>
        <width>551</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tin tá»©https://baomoi.com/doi-hinh-singapore-vs-viet-nam-nguyen-filip-du-bi-xuan-son-da-chinh-c51092134.epic 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>320</y>
        <width>521</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>https://baomoi.com/singapore-vs-viet-nam-co-hoi-noi-dai-mach-bat-bai-c51092124.epi</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>560</y>
        <width>501</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>https://baomoi.com/cuu-trong-tai-chung-ket-aff-cup-2008-nho-ve-chien-thang-cua-viet-nam-c51091962.epi</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(155, 232, 232);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>5:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>141</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>T6 20/12</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Thá»i gian</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(155, 232, 232);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>6:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(155, 232, 232);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>7:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>130</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(155, 232, 232);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>8:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(155, 232, 232);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>9:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>130</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(155, 232, 232);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>10:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>130</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(155, 232, 232);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>11:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>130</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(155, 232, 232);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>13:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>130</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(155, 232, 232);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>12:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>130</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(155, 232, 232);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>14:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>130</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(155, 232, 232);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>15:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>SÃ¢n 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>SÃ¢n 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>SÃ¢n 3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>SÃ¢n 4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>410</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>SÃ¢n 5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>460</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>SÃ¢n 6</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>510</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>SÃ¢n 7</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>560</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>SÃ¢n 8</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>610</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>SÃ¢n 9</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>660</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>SÃ¢n 10</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="nav_home_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/house-solid.svg</normaloff>../img/house-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="nav_new_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/newspaper-solid.svg</normaloff>../img/newspaper-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="nav_account_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/user-solid.svg</normaloff>../img/user-solid.svg</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
